src/main/java/com/hillel/items_exchange/service/CategoryService.java

```diff
@@ -1,8 +1,25 @@
 package com.hillel.items_exchange.service;
 
+import com.hillel.items_exchange.dto.CategoryControllerDto;
+
 import java.util.List;
+import java.util.Optional;
 
 public interface CategoryService {
```

Comment:
I don't really understand why do we need this interface instead of using implementation directly?